Source/Basic-Conditions-And-Behaviors/Runtime/Conditions/CollectedCondition.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using VRBuilder.Core.Attributes;
+using VRBuilder.Core.SceneObjects;
+using UnityEngine;
+using VRBuilder.Core.Properties;
+
+namespace VRBuilder.Core.Conditions
+{
+    /// <summary>
+    /// A condition that checks if new selected objects are entering the selected collider.
+    /// </summary>
+    [DataContract(IsReference = true)]
+    public class CollectedCondition: Condition<CollectedCondition.EntityData>
+    {
+        [DataContract(IsReference = true)]
+        [DisplayName("Collect Condition")]
+        public class EntityData: IConditionData
+        {
+            [HideInProcessInspector]
+            public readonly List<GameObject> Collected = new();
+
+            /// <summary>
+            /// The objects that has to enter the collider.
```

Comment:
```suggestion
            /// The objects that can be placed in the collider.
```
Sounds clearer to me.